src/index.js

```diff
@@ -98,10 +172,45 @@ function listTopMovies(input) {
         .addClass('.error')
         .text(messages[Error.NO_RESULTS])
         .fadeIn('slow');
+      delayFlashHide();
     });
 }
 
-function fetchMovies(year) {
-  let url = urlJoin(proxyUrl, baseUrl, '/year/', year);
-  return fetchDocument(url).then(doc => parseMovies(doc));
+/* infinite scroll */
+
+$window.scroll(e => {
+  if ($(e.target).scrollTop()) {
+    $arrowToTop.fadeIn();
+  } else {
+    $arrowToTop.fadeOut();
+  }
+  if(($window.scrollTop() === $document.height() - $window.height()) && pagesFlag) {
+    query.page++;
+    listTopMovies(query);
+  }
+});
+
+/* scroll to top */
+
+$arrowToTop.click(() => {
+  $html$body.animate({ scrollTop: 0 }, 'slow');
+  return false;
+});
+
+
+/* large img display */
+
+$movieList.on('click', movieImgSmall, (e) => {
+  let srcStr = $(e.target).attr('src');
+  displayLargeImg(srcStr);
+  e.preventDefault();
+});
+
+function displayLargeImg(src) {
+  if(!src || !(src.match(/@{0,}(\._V1.*)\.jpg$/))){
+    return ;
+  }
+  let str = src.match(/@{0,}(\._V1.*)\.jpg$/)[1];
```

Comment:
Ok, so you have following url:
https://images-na.ssl-images-amazon.com/images/M/MV5BMTc1OTc4MTQ5MF5BMl5BanBnXkFtZTgwMTQ0MTM4MTI@._V1_UX67_CR0,0,67,98_AL_.jpg
and you want convert it to:
https://images-na.ssl-images-amazon.com/images/M/MV5BMTc1OTc4MTQ5MF5BMl5BanBnXkFtZTgwMTQ0MTM4MTI@.jpg

Simple way:
```js
let url = src.replace(/\._V(.*)$/, '.jpg');
```